Sources/Sentry/SentryFileManager.m

```diff
@@ -751,12 +755,32 @@ - (void)createPathsWithOptions:(SentryOptions *)options
     return sentryLaunchConfigFileURL;
 }
 
+NSURL *
+launchProfileConfigBackupFileURL(void)
+{
+    static NSURL *sentryLaunchConfigFileURL;
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+        sentryLaunchConfigFileURL =
+            [[NSURL fileURLWithPath:[sentryApplicationSupportPath()
+                                        stringByAppendingPathComponent:@"profileLaunch"]]
+                URLByAppendingPathExtension:@"bak"];
+    });
+    return sentryLaunchConfigFileURL;
+}
+
 NSDictionary<NSString *, NSNumber *> *_Nullable appLaunchProfileConfiguration(void)
 {
     NSError *error;
-    NSDictionary<NSString *, NSNumber *> *config = [NSDictionary<NSString *, NSNumber *>
-        dictionaryWithContentsOfURL:launchProfileConfigFileURL()
-                              error:&error];
+    NSURL *configFileURL;
+    if (isTracingAppLaunch) {
```

Comment:
`l`: It would be better to pass that in as a parameter to specify if you want the backup or the current config. This would give us more flexibility, and it communicates more clearly what it returns. Otherwise, you have to know that the global state `isTracingAppLaunch` changes the behavior of that method.